images/krte/Dockerfile

```diff
@@ -26,7 +26,7 @@ ARG GO_VERSION
 # - adding gsutil etc. to path (where we will install them)
 # - disabling prompts when installing gsutil etc.
 # - hinting that we are in a docker container
-ENV IMAGE=${IMAGE_ARG} \
+ENV KRTE_IMAGE=${IMAGE_ARG} \
```

Comment:
kubekins should also do that. will leave to another PR.
(`IMAGE` is commonly used in makefiles etc.)